storage/rocksdb/ha_rocksdb.cc

```diff
@@ -417,6 +433,8 @@ static char *rocksdb_strict_collation_exceptions;
 static my_bool rocksdb_collect_sst_properties = 1;
 static my_bool rocksdb_force_flush_memtable_now_var = 0;
 static my_bool rocksdb_force_flush_memtable_and_lzero_now_var = 0;
+static my_bool rocksdb_enable_ttl = 1;
```

Comment:
Sure enough, my persistent_cache test instability is gone by fixing up the variable inits. This is a very good and worthwhile effort.